code/__HELPERS/spatial_info.dm

```diff
@@ -118,6 +118,45 @@
 
 	return .
 
+/**
+ * The exact same as get_hearers_in_view, but not limited by visibility. Does no filtering for traits, line of sight, or any other such criteria.
+ * Filtering is intended to be done by whatever calls this function.
+ *
+ * This function exists to allow for mobs to hear speech without line of sight, if such a thing is needed.
+ *
+ * * view_radius - what radius search circle we are using, worse performance as this increases
+ * * source - object at the center of our search area. everything in get_turf(source) is guaranteed to be part of the search area
+ */
+/proc/get_hearers(view_radius, atom/source)
```

Comment:
should be get_hearers_in_range(), also the first argument should be just range or radius